kapitan/refs/base.py

```diff
@@ -435,6 +471,24 @@ def token_type_name(self, token):
 
         return type_name
 
+    def ref_from_embedded(self, type_name, b64_path):
+        "returns ref from embedded (base64 and json) b64_path"
+        # deserialise base64 and json data from b64_path
+        json_data = base64.b64decode(b64_path).decode()
+        json_data = json.loads(json_data)
+        backend = self._get_backend(type_name)
+        # strip useless keys
+        json_data.pop("encoding")
+        json_data.pop("type")
+
+        data = json_data.pop("data").encode()
+        # create new ref with deserialised data and remaining keys as kwargs
+        # note that encrypt=False is only for secret ref types, others will ignore
```

Comment:
This comment is good, but the sentence after ", others" could be slightly clearer?